components/service/sync-logins/README.md

```diff
@@ -161,6 +161,12 @@ Currently there is no way to change the key, once set (see https://github.com/mo
 
 Part is in this tree, but most is in https://github.com/mozilla/application-services. Specifically, there are two pieces not present in this repository, an [android-specific piece written in Kotlin](https://github.com/mozilla/application-services/tree/master/logins-sql/tree/master/logins-api/android), and a [cross-platform piece written in Rust](https://github.com/mozilla/application-services/tree/master/logins-sql).
 
+### What's the difference between `reset`, `wipe` and `wipeLocal`?
```

Comment:
Thanks for documenting these!

Reply:
Okay, I'm keeping the mention of `reset` to make it clear that it's not the case that reset was renamed to wipeLocal (while being very clear that it no longer exists).